src/Microsoft.ML.Data/DataLoadSave/CompositeDataLoader.cs

```diff
@@ -1,616 +1,118 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Collections.Generic;
 using System.IO;
-using System.Linq;
 using Microsoft.Data.DataView;
-using Microsoft.ML;
-using Microsoft.ML.CommandLine;
-using Microsoft.ML.Data;
-using Microsoft.ML.Internal.Utilities;
 using Microsoft.ML.Model;
 
-[assembly: LoadableClass(typeof(IDataLoader), typeof(CompositeDataLoader), typeof(CompositeDataLoader.Arguments), typeof(SignatureDataLoader),
-    "Composite Data Loader", "CompositeDataLoader", "Composite", "PipeData", "Pipe", "PipeDataLoader")]
-
-[assembly: LoadableClass(typeof(IDataLoader), typeof(CompositeDataLoader), null, typeof(SignatureLoadDataLoader),
-    "Pipe DataL Loader", CompositeDataLoader.LoaderSignature)]
```

Comment:
Why remove the entrypoints? Shouldn't these remain?

Reply:
I see -- this is moved to LegacyCompositeDataLoader.

---
In reply to: [260564833](https://github.com/dotnet/machinelearning/pull/2731#discussion_r260564833) [](ancestors = 260564833)

Reply:
Would it better to git rename CompositeDataLoader -> LegacyCompositeDataLoader and then add a new file of CompositeDataLoader?

---
In reply to: [260565188](https://github.com/dotnet/machinelearning/pull/2731#discussion_r260565188) [](ancestors = 260565188,260564833)

Reply:
Will do thank you for pointing it out and taking another look at this. 

---
In reply to: [260565382](https://github.com/dotnet/machinelearning/pull/2731#discussion_r260565382) [](ancestors = 260565382,260565188,260564833)